src/Commands/ContinuousIntegration/TravisCi/TravisCiCommandFactory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace QuickStrap\Commands\ContinuousIntegration\TravisCi;
+
+use QuickStrap\Commands\ContinuousIntegration\TravisCi\Config\ConfigBuilder;
+use QuickStrap\Commands\ContinuousIntegration\TravisCi\Config\ConfigWriter;
+use QuickStrap\Helpers\Composer\PackageHelper;
+use Symfony\Component\Console\Helper\HelperSet;
+use Symfony\Component\Console\Helper\QuestionHelper;
+
+class TravisCiCommandFactory
+{
+    /**
+     * @param HelperSet $helpers
+     * @return TravisCiCommand
+     * @throws \Symfony\Component\Console\Exception\InvalidArgumentException
+     * @throws \Symfony\Component\Console\Exception\LogicException
+     */
+    public static function createCommand(HelperSet $helpers)
+    {
+        $questionHelper = self::getQuestionHelper($helpers);
+        
+        $travisCiQuestionHelper = new TravisCiQuestionHelper($questionHelper);
+
+        $packageHelper = self::getPackageHelper($helpers);
+        
+        $configBuilder = new ConfigBuilder($travisCiQuestionHelper, $packageHelper);
+
+        $configWriter = new ConfigWriter();
+
+        return new TravisCiCommand($travisCiQuestionHelper, $configBuilder, $configWriter);
+    }
+
+    /**
+     * @param HelperSet $helpers
+     * @return QuestionHelper
+     * @throws \Symfony\Component\Console\Exception\InvalidArgumentException
+     */
+    private static function getQuestionHelper(HelperSet $helpers)
+    {
+        return $helpers->get('question');
+    }
+
+    /**
+     * @param HelperSet $helpers
+     * @return PackageHelper
+     * @throws \Symfony\Component\Console\Exception\InvalidArgumentException
+     */
+    private static function getPackageHelper(HelperSet $helpers)
```

Comment:
Not sure how this is helpful. Replacing one line with another, and doesn't help for testing.